resources/extruders/trimaker_nebula_extruder.def.json

```
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "Trimaker Nebula",
```

```suggestion
        "machine": "trimaker_nebula",
```
It needs to reference to the ID of the machine, not the name.